Projeto_Tkinter/Control.py

```diff
@@ -40,16 +43,17 @@ def Puxa_dados(self):
         return dados
 
     def Func_Validar_user(self):
-        dados = [[],[]]
-        dados[0] = 'david'; dados[1] = '123'
-        if not dados[0] and not dados[1]:
-            self.create.Info_Labls_Login_Erro()
+        dados = self.Puxa_dados()
+        dados['nome'] = 'david'; dados['senha'] = '123'
+        if not dados['nome'] and not dados['senha']:
```

Comment:
```suggestion
        if not dados['nome'] or not dados['senha']:
```
Entendo que se um dos dois não está presente, ele deve disparar o erro, não?